tree/tree/src/TBranch.cxx

```diff
@@ -1163,6 +1163,9 @@ TBasket* TBranch::GetBasket(Int_t basketnumber)
          }
       }
       Error("GetBasket","File: %s at byte:%lld, branch:%s, entry:%lld, badread=%d, nerrors=%d, basketnumber=%d",file->GetName(),basket->GetSeekKey(),GetName(),fReadEntry,badread,nerrors.load(),basketnumber);
+      if (basket->IsZombie()) {
+         MakeZombie();
```

Comment:
This is too strong of statement.  In the usual case, there is some corruption on the file and **some** but not all the basket can be read.